backend/index.js

```diff
@@ -9,6 +9,8 @@ const prisma = new PrismaClient();
 const app = express();
 const PORT = process.env.PORT || 3000;
 
+const requests = 70; 
```

Comment:
nit: for global consts like this the general naming convention is upper snake case
```
const MAX_REQUESTS_PER_API_LIMIT = 70
```